src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs

```diff
@@ -1186,4 +1186,31 @@ public bool Value
             }
         }
     }
+
+    /// <summary>
+    /// EventArgs type for the RotaryEventEnabledChanged event.
+    /// </summary>
+    /// <privilege>http://tizen.org/privilege/systemsettings.admin</privilege>
+    /// <privlevel>platform</privlevel>
+    /// <feature>http://tizen.org/feature/systemsetting</feature>
+    /// <exception cref="NotSupportedException">Not Supported feature</exception>
+    /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+    /// <remarks>
+    /// http://tizen.org/privilege/systemsettings.admin is needed only for setting value. When getting the value, it isn't needed.
+    /// </remarks>
+    /// <since_tizen> 6 </since_tizen>
+    public class RotaryEventEnabledChangedEventArgs : EventArgs
+    {
+        private readonly bool _rotaryEventEnabled;
+        internal RotaryEventEnabledChangedEventArgs(bool val)
+        {
+            _rotaryEventEnabled = val;
```

Comment:
```suggestion
            Value = val;
``` 
and remove _rotaryEventEnabled